### PR TITLE
Upgrade provider to `~>4.1`

### PR DESCRIPTION
**Pull request by soerenmartius**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Set minimum required provider version to `v4.1`
+
+### Changed
+
+- Delete `resource "google_project_service` resource
```
Comment (mariux) [inline CHANGELOG.md:16]:
```suggestion
- Remove support for `google_project_service` resource
```

**Review by mariux — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (soerenmartius) [reply CHANGELOG.md]: resolved in 4128978d

Comment (soerenmartius) [reply CHANGELOG.md]: resolved in 4128978d

**Review by thiesen — APPROVED** [review]
lgtm!

**Pull request merged**